### ISSUE TITLE
markdownlint_docker pre-commit fails

### ISSUE THREAD

**Issue by OtherDevOpsGene** [markdownlint/markdownlint#363]
## Description
When using the pre-commit hook `markdownlint_docker`, the pre-commit hook errors out because the file `mdl` to lint cannot be found.

### Environment
Windows 10 PowerShell and WSL2 Ubuntu on the same system show the same behavior.
**MDL Version**
0.11.0

### Expected Behavior
The Markdownlint Docker pre-commit hook should run successfully.

### Actual Behavior
The pre-commit hook errors because it can't find the file `mdl` to scan.

```
Markdownlint Docker......................................................Failed
- hook id: markdownlint_docker
- exit code: 1

/usr/lib/ruby/gems/2.6.0/gems/mdl-0.11.0/lib/mdl/doc.rb:50:in `read': No such file or directory @ rb_sysopen - mdl (Errno::ENOENT)
        from /usr/lib/ruby/gems/2.6.0/gems/mdl-0.11.0/lib/mdl/doc.rb:50:in `new_from_file'
        from /usr/lib/ruby/gems/2.6.0/gems/mdl-0.11.0/lib/mdl.rb:82:in `block in run'
        from /usr/lib/ruby/gems/2.6.0/gems/mdl-0.11.0/lib/mdl.rb:80:in `each'
        from /usr/lib/ruby/gems/2.6.0/gems/mdl-0.11.0/lib/mdl.rb:80:in `run'
        from /usr/lib/ruby/gems/2.6.0/gems/mdl-0.11.0/bin/mdl:10:in `<top (required)>'
        from /usr/bin/mdl:23:in `load'
        from /usr/bin/mdl:23:in `<main>'
```

## Replication Case

If the file `README.md` was the only file being committed, then the Docker command run by pre-commit would be

```bash
docker run -v $PWD:/src:rw,Z --workdir /src markdownlint/markdownlint mdl mdl README.md
```

which would try to run against `/src/mdl` and `/src/README.md`, neither of which exists.

* `v $PWD:/src:rw,Z --workdir /src` is implicit in pre-commit for a `docker_image` hook
* the first `mdl` is the `ENTRYPOINT` from the Dockerfile, so it is the command to run inside the Docker container
* the second `mdl` is the `entry` from `.pre-commit-hooks.yaml`, which looks like the first file to scan to the Docker image
* `README.md` is the only file to scan passed by pre-commit, which looks like the second file to scan to the Docker image

**Comment by LukeBoylePM** [comment]
**
![Uploading Screenshot 2023-08-24 at 10.30.54.png…]()
**

**Comment by jaymzh** [comment]
@princeraphaelmomoh - this is a very old bug that was closed. If you're experiencing a similar problem please file a new issue.
